rasa/core/channels/twilio_voice.py

```diff
@@ -303,6 +328,15 @@ def _build_twilio_voice_response(
                 voice_response.pause(length=1)
 
         return voice_response
+    
+    async def _twilio_record(self,call_sid):
+        if not self.record:
+            return
+        url= f"https://api.twilio.com/2010-04-01/Accounts/{self.account_sid}/Calls/{call_sid}/Recordings.json"
+        obj={}
+        await asyncio.sleep(1)
```

Comment:
Does this imply that the first second of the call will not be recorded? Can you help me understand why was this required and if you experimented with any values less than 1 second?

Reply:
The full call is recorded. I did not see any part of the call being missed. The delay was needed because twilio gave errors otherwise - its like the call did not yet exist on twilios system